docs/src/main/tut/generate-sources-from-idl/avro.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: docs
+title: Generating sources from Avro
+permalink: /generate-sources-from-avro
+---
+
+# Generating sources from Avro
+
+In this section we are going to explain how we can generate the different Scala structures using the `Avro` IDL.
+
+## Protocols
+
+Let's start from the beginning, everything on `Avro` should be declared inside a `protocol`. 
+
+The name of that protocol will be the name of our Scala file.
+
+```avroidl 
+protocol People { 
+ ... 
+} 
+```
+
+***srcGen =>***
```

Comment:
Maybe it worth mentioning that `srcGen` is an sbt command that is run (by default) in the `compile` phase